src/index.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import changeCase from 'case';
 import styleParser from 'style-attr';
 
+function isFunction(thing) {
+  return Object.prototype.toString.call(thing) === '[object Function]';
+}
+
+function getAframeEventMappings(possibleEventHandlers) {
+  return Object.keys(possibleEventHandlers)
+    .filter(reactEventName => (
+      !!reactEventName.match(/^on[A-Z]/))
+      && isFunction(possibleEventHandlers[reactEventName])
+    ).reduce((handlers, reactEventName) => {
+      const aframeEventName = changeCase.kebab(reactEventName.replace(/^on/, ''));
```

Comment:
Could that lead to possibly ambiguous situations in the future? Maybe someone comes up with a new component and wants to fire a `mouse-enter` event knowing that `mouseenter` is already taken (naming things is hard)? Now both events are handled, when we really only wanted one.